src/core/filters/filterregistry.js

```diff
@@ -57,18 +56,65 @@ export default class FilterRegistry {
   }
 
   /**
-   * Gets the filter string to send in a search query.
-   * TODO: move payload method logic into core.js, since it is only used there.
-   * @returns {string}
+   * Gets the static filters as a {@link CombinedFilter} to send to the answers-core
    */
   getStaticFilterPayload () {
-    return JSON.stringify(this._getStaticFilterPayload());
+    const filterNodes = this.getStaticFilterNodes();
+
+    const transformedFilterNodes = filterNodes.map(filterNode => {
+      return this._transformFilter(filterNode);
+    }).filter(filter => filter);
+
+    if (transformedFilterNodes.length > 1) {
+      return {
+        combinator: '$and',
+        filters: transformedFilterNodes
+      };
+    }
+    return transformedFilterNodes[0];
```

Comment:
Do we need to consider the case of `filterNodes.length === 0`?

Reply:
In that case, the `transformedFilterNodes = []` so this function would return undefined. Is that expected, or should we add in alternate fallback behavior?

Reply:
This is ultimately passed to the Core library, right? I'm assuming the library would interpret `undefined` as no static filters and so none would be included in the request.

Reply:
Yes exactly!

Reply:
I think we should at least make it clear in the JSDoc that this method may return `undefined`. Someone reading through this code may be a bit confused to see an `[0]` accessor, without a check that `length === 1`.